cxx-qt-cli/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2021 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+# SPDX-FileContributor: Be Wilson <be.wilson@kdab.com>
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+[package]
+name = "cxx-qt-cli"
+authors = ["Be Wilson <be.wilson@kdab.com>"]
+version = "0.3.0"
+edition = "2021"
+description = "Command line tool for cxx-qt's C++ code generation step for integration into C++ build systems"
+repository = "https://github.com/KDAB/cxx-qt/"
+
+[[bin]]
+name = "cxxqtbridge"
+path = "src/main.rs"
+
+[dependencies]
+cxx-qt-build = { path = "../cxx-qt-build" }
+clap = { version = "3.2.12", features = [ "derive" ] }
+
+[features]
+# Requires nightly compiler until std::path::absolute is stabilized
```

Comment:
That's why this is behind a feature that is disabled by default.

Reply:
Ah i missed that it is shadowing the `output_file_path` i thought it might fail without the feature :') ... So then this is fine maybe just put a `TODO: FUTURE:` or something so we can find it later when / if the feature becomes stable.